conf/application.conf

```diff
@@ -238,6 +238,10 @@ voa {
       detailedValuation.path = "/external-valuation-management-api/properties/{uarn}/valuations/{valuationId}"
       valuationLookupUrl.path = "/external-valuation-management-api/valuationLookups/{lookupType}"
     }
+
+    customerManagementApi {
+      getAgentByRepresentationCode.path = "/customer-management-api/organisation?representativeCode={agentCode}"
```

Comment:
dont add query parameters to the URL, http-verbs gives you the ability to add query parameters to the url with the Seq[(String, String)]. this makes us more consistent with the platform and allows more query parameters to be added later without changing the base url.